Samples/AudioPreprocessing/AudioPreprocessing/Model/PreprocessModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AudioPreprocessing.Model
+{
+    public class PreprocessModel
+    { 
+        private List<string> repositoryData;
+        private string melSpecPath;
+        public string ImportantInfo
+        {
+            get
+            {
+                return ConcatenateData(repositoryData);
+            }
+        }
+        public string MelSpecPath
+        {
+            get
+            {
+                return melSpecPath;
+            }
+        }
+
+        public PreprocessModel()
+        {
+            repositoryData = GetData();
+            melSpecPath = "C:/Users/t-janedu/source/repos/Windows-Machine-Learning/Samples/AudioPreprocessing/AudioPreprocessing/tmp/mel_spectrogram_file.jpg";
```

Comment:
can this be a relative path instead of something specific to your machine?

Reply:
Corrected this!